polaris/deploy/roles/jasper_reports/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+jasper_server_instance: "{{ jasper_server_instance | upper | default('') }}"
+jasper_project_name: "{{ jasper_project_name | upper | default('') }}"
+jasper_ds_0_url: "{{ lookup('ansible.builtin.env', 'PODMAN_cfg_jasper_ds_0_url') | default('') }}"
```

Comment:
I'm leaning towards banning env lookups from the collection. These would just then be blank and be something the composer could setup.